tests/integration_tests/clouds.py

```diff
@@ -44,6 +47,11 @@ def _get_initial_image(self):
             pass
         return image_id
 
+    def _perform_launch(self, launch_kwargs):
```

Comment:
Maybe we could rename that method to _ create_instance and instead just call here the `init` method from cloudinit.
We could them move the calls:

```
 pycloudlib_instance.start(wait=False)
 pycloudlib_instance.wait(raise_on_cloudinit_failure=False)
```

back into the base `launch` method.

Even though we would not be using the default api of pycloudlib, since pycloudllib launch already performs the `start` operation, I think that those changes will improve a little bit the naming of the new method

In summary, turn the `_perform_launch` into the methods:

```python
class IntegrationCloud(ABC):
    def _create_instance(self, launch_kwargs):
        return self.cloud_instance.init(**launch_kwargs)

class LxdContainerCloud(IntegrationCloud):
    def _create_instance(self, launch_kwargs):
        launch_kwargs['inst_type'] = launch_kwargs.pop('instance_type', None)
        launch_kwargs.pop('wait')

        pycloudlib_instance = self.cloud_instance.init(
            launch_kwargs.pop('name', None),
            launch_kwargs.pop('image_id'),
            **launch_kwargs
        )
        if self.settings.CLOUD_INIT_SOURCE == 'IN_PLACE':
            self._mount_source(pycloudlib_instance)

        return pycloudlib_instance
```

Reply:
Problem is that `init` only lives in the LXD cloud, not in all of the clouds, so `return self.cloud_instance.init(**launch_kwargs)` wouldn't work in our base class.